balancer-js/src/modules/sor/sor.module.ts

```diff
@@ -99,7 +99,6 @@ export class Sor extends SOR {
         network.addresses.tokens.wrappedNativeAsset
       );
     }
-
-    return new CoingeckoTokenPriceService(network.chainId);
+    return new ApiTokenPriceService(network.chainId);
```

Comment:
Right above this, there's a check for `tokenPriceService` set by the `sorConfig`. Can you please add a new option there called `api` and make it the default one within `getSorConfig`?
There are also several tests within the SDK that use coingecko as the tokenPriceService - maybe a nice check would be to update those to use the api and see how they behave.

Reply:
Right above these lines, there's a check for each `tokenPriceService`. It would be nice to add one for `coingecko` in case anyone wants to use it.